src/View/JsonApiView.php

```diff
@@ -213,8 +213,27 @@ protected function _entitiesToNeoMerxSchema(array $entities)
                 throw new CrudException('JsonApiListener cannot not find Entity class ' . $entityName);
             }
 
+            $entityShortName = $entityName;
+            if (strpos($entityName, '\\') !== false) {
```

Comment:
The entity names that come from the `entiyClass` method are actual full class names. This block of code is necessary to turn those full class names back into the plugin split format which is required by `App::className`.